crates/agent/src/handlers.rs

```diff
@@ -113,16 +116,39 @@ enum Status {
     Idle,
 }
 
+const PUBLICATIONS_TABLE: &str = "publications";
+const CONTROLLER_JOBS_TABLE: &str = "controller_jobs";
+
+/// The purpose of this is to ensure that the controllers handler gets polled
```

Comment:
I understand our intention is to rip all of this out when we tackle the updated lease-based job queue model.

We'll still want a mechanism to immediately wake a job dequeuer from a polling backoff sleep though :thinking:

Reply:
Yeah, one thing I realized is that our existing notifications mechanism doesn't really work for `controller_jobs` because they don't have a `{"type":"queued"}` status, so they rely on the periodic triggers from within the agent, as well as this hack. But I think that a periodic trigger seems totally fine, as long as we can also trigger controllers promptly in response to a publication. The main improvements I'd like to see here are to make the periodic trigger more explicit and targeted to controllers, and to make this publications hack somewhat less "hacky" (perhaps :wave: by passing a `ControllersWaker` into the `Publisher`).